be/src/olap/options.cpp

```diff
@@ -17,26 +17,36 @@
 
 #include "olap/options.h"
 
-#include <boost/algorithm/string.hpp>
-#include <boost/algorithm/string/classification.hpp>
-#include <boost/algorithm/string/split.hpp>
-
 #include "common/logging.h"
 
 #include "olap/utils.h"
 
+#include "gutil/strings/split.h"
+
 namespace doris {
 
+static std::string CAPACITY = "CAPACITY";
+static std::string MEDIUM = "MEDIUM";
+static std::string SSD = "SSD";
+static std::string HDD = "HDD";
+
+// Modify in place
+// TODO: should be a general util method
+std::string to_upper(std::string* str) {
```

Comment:
If you intent to modify in place, there is no need to create another string to return. However it is better to create a new string other than to modify in place. So I suggest you change it to 
```suggestion
std::string to_upper(const std::string& str) {
```

Reply:
Done.